events-to-histogram/default.nix

```diff
@@ -20,37 +20,4 @@
       GIT_REVISION = git_revision;
     };
   };
-
-  container-image = let
```

Comment:
We still want the container images, they just do not need the entrypoint script.

See https://github.com/DanNixon/satori/blob/main/archiver/default.nix#L24 for an example.